CMakeLists.txt

```diff
@@ -573,7 +575,8 @@ if(AVIF_BUILD_TESTS)
         set_target_properties(avify4mtest PROPERTIES LINKER_LANGUAGE "CXX")
     endif()
     target_link_libraries(avify4mtest avif ${AVIF_PLATFORM_LIBRARIES})
-    target_include_directories(avify4mtest PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
+    target_include_directories(avify4mtest
+        PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/apps/shared)
```

Comment:
Nit: `${CMAKE_CURRENT_SOURCE_DIR}/` can be omitted. Please compare with line 559. We should use the same approach at both places.

I am fine with either approach, but slightly prefer the more concise approach unless it is broken in some build configurations.

Reply:
Indeed, from [CMake doc](https://cmake.org/cmake/help/latest/command/target_include_directories.html):

> A relative path will be interpreted as relative to the current source directory (i.e. CMAKE_CURRENT_SOURCE_DIR)